test/setup.ts

```diff
@@ -0,0 +1,8 @@
+import { chai, sinon } from './utils';
+import * as sinonChai from 'sinon-chai';
+
+chai.use(sinonChai);
+
+afterEach(() => {
+  sinon.restore();
```

Comment:
I believe sinon.restore() is deprecated. People are now encouraged to use sandboxes.

https://github.com/sinonjs/sinon/issues/1390

Reply:
Starting with sinon v5.0, sinon itself becomes the default sandbox. 

https://sinonjs.org/guides/migrating-to-5.0.html